sklearn/linear_model/tests/test_coordinate_descent.py

```diff
@@ -458,7 +473,14 @@ def test_linear_model_sample_weights_normalize_in_pipeline(
     # linear estimator with built-in feature normalization
     reg_with_normalize = estimator(normalize=True, fit_intercept=True,
                                    **params)
-    reg_with_normalize.fit(X_train, y_train, sample_weight=sample_weight)
+
+    if model_name in ['Lasso', 'ElasticNet']:
+        ctxmgr = pytest.warns(ConvergenceWarning)
+    else:
+        ctxmgr = contextlib.suppress()
```

Comment:
It might be good to have a comment to mention that `supress()` without parameters will not suppress warnings :)

Reply:
Indeed! I would liked to use Python >= 3.7 contextlib.nullcontext, but that is not supported on 3.6 it seems. Will update in a minute.

Reply:
Oh indeed, we will support only `Python 3.7` in version 1.0 so we could be proactive. I don't remember if we move all our CI from 3.6 to 3.7

Reply:
For now I added a comment to the first instance of `suppress()`, and mentioned also Python 3.7 there. If all is switched we can use nullcontext (which would somehow be *nicer*)

Reply:
@glemaitre all the CI has been moved to python 3.7.

Reply:
Great, so I have switched to use `nullcontext()` - let us see how it goes ;-)